added link for the examples of envFrom (#40579)
kubernetes/website: content/en/docs/tasks/inject-data-application/define-environment-variable-container.md

   DEMO_FAREWELL=Such a sweet sorrow
   ```

### Setting container environment variables with `envFrom`

In this exercise, you create a Pod that runs one container.
But first we need to reference it to a ConfigMap 

{{< codenew file="pods/inject/configmap-envFrom.yaml" >}}

Now as the kind: ConfigMap is created we need to 
reference it to a pod. 

{{< codenew file="pods/inject/envFrom.yaml" >}}

You can also use Secret instead of ConfigMap for the configuration

> ```

Reviewer: nit: could append `yaml` here for syntax highlighting.

Author: done

apiVersion: v1
kind: Secret
metadata:
  name: mysecret
type: Opaque
data:
  DB_HOST: bXlkb2Joc3RhcnQ= # Base64-encoded value for mydbhost
  DB_PORT: NTQzMiA= # Base64-encoded value for 5432
  DB_NAME: bXlkYm5hbWU= # Base64-encoded value for mydbname

```

> Now as the Kind: Secret is created we need to reference it to a pod.

Reviewer: ```suggestion
If you created that Secret, you could reference it in a bare Pod:
```

```
apiVersion: v1
kind: Pod
metadata:
  name: mypod
spec:
  containers:
  - name: mycontainer
    image: myimage
    envFrom:
    - secretRef:
        name: mysecret
```

> 

Reviewer: ```suggestion

## Interactions between environment variables

The environment variables you set using the `env` or `envFrom` field
override any environment variables specified in the container image.

Environment variables may reference each other, however ordering is important.
Variables making use of others defined in the same context must come later in
the list. Similarly, avoid circular references.
```

> {{< note >}}

Reviewer: Try replacing the two existing notes here with:
```markdown

## Interactions between environment variables

The environment variables you set using the `env` or `envFrom` field
override any environment variables specified in the container image.

Environment variables may reference each other, however ordering is important.
Variables making use of others defined in the same context must come later in
the list. Similarly, avoid circular references.
```
